src/test/java/org/dependencytrack/resources/v1/ProjectResourceTest.java

```diff
@@ -2517,19 +2473,16 @@ public void issue3883RegressionTest() {
                       "version": "1.0.0",
                       "classifier": "APPLICATION",
                       "uuid": "${json-unit.any-string}",
-                      "active": true,
                       "isLatest": false
                     }
                   ],
                   "properties": [],
                   "tags": [],
-                  "active": true,
                   "isLatest": false,
                   "versions": [
                     {
                       "uuid": "${json-unit.any-string}",
-                      "version": "1.0.0",
-                      "active": true
+                      "version": "1.0.0"
```

Comment:
The `active` flag is required by the frontend to visually separate active from inactive project versions in the version dropdown: https://github.com/DependencyTrack/hyades-frontend/blob/5094b723e9b09e6f6d1bf4b2ca374b4e25299e61/src/views/portfolio/projects/Project.vue#L42-L67

By not returning this flag, we'd be breaking said frontend feature.